content/en/Platform Deep Dive/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Platform Deep Dive"
+linkTitle: "Platform Deep Dive"
+weight: 10
+toc_hide: true
+description: >
+  Explore the features of the Cobalt platform.
+---
+
+{{% pageinfo %}}
+We're working to expand this section of our Product Documentation. Stay tuned.
```

Comment:
Normally I dislike this kind of "We're doing something in the future" kind of statement. 

But we are -- and we have to, in order to complete the process.